Formula/edbrowse.rb

```diff
@@ -0,0 +1,84 @@
+class Edbrowse < Formula
+  desc "Command-line editor and web browser"
+  homepage "https://www.edbrowse.org"
+  url "https://github.com/CMB/edbrowse/archive/v3.8.2.tar.gz"
+  sha256 "fed3d2a2ffa5f296ed5edeb5e08ac4cf0f4aec55c8c50945e3f45e300dd3706e"
+  license all_of: ["GPL-2.0-only", "MIT"]
+  head "https://github.com/CMB/edbrowse.git", branch: "master"
+
+  depends_on "make" => :build
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pcre2"
+  depends_on "readline"
+  depends_on "tidy-html5"
+  depends_on "unixodbc"
+  uses_from_macos "curl"
+
+  resource "quickjs" do
+    url "https://github.com/bellard/quickjs/archive/b5e62895c619d4ffc75c9d822c8d85f1ece77e5b.tar.gz"
+    sha256 "640fb75b3ca11eec37f9df4bea0afa48a39027ca2dc60806b96d29aa03e504a6"
+  end
+
+  def install
+    resource("quickjs").stage do
```

Comment:
I don't suspect quickjs would accept this patch. Edbrowse patches it with a shell script, but I am doing it here to avoid more dependencies. If you'd prefer, I could see if they can maintain a forked quickjs, and I can pull that one in, if that would be okay?

Reply:
Upstream maintaining a fork sounds like a better solution indeed.